pkg/services/ngalert/store/image.go

```diff
@@ -36,6 +40,19 @@ func (st DBstore) GetImage(ctx context.Context, token string) (*models.Image, er
 	return &img, nil
 }
 
+func (st DBstore) GetImages(ctx context.Context, tokens []string) ([]models.Image, error) {
+	var imgs []models.Image
+	if err := st.SQLStore.WithDbSession(ctx, func(sess *sqlstore.DBSession) error {
+		return sess.In("token", tokens).Find(&imgs)
+	}); err != nil {
+		return nil, err
+	}
+	if len(imgs) < len(tokens) {
```

Comment:
Let's document somewhere that ErrImageNotFound can indicate partial results. I think it doesn't require a new error type.

Reply:
It's documented on `GetImages` for the `ImageStore` interface